src/functions/transaction-to-webhook/handler.ts

```diff
@@ -1,8 +1,14 @@
 import { type CoreTransactionProcessedEvent } from "@stedi/integrations-sdk";
-import { bucketsClient } from "@stedi/integrations-sdk/clients";
+import { bucketsClient, mappingsClient } from "@stedi/integrations-sdk/clients";
 import { GetObjectCommand } from "@stedi/sdk-client-buckets";
+import { MapDocumentCommand } from "@stedi/sdk-client-mappings";
 
 export const handler = async (event: CoreTransactionProcessedEvent) => {
+  // fail fast if WEBHOOK_URL env var is not defined
```

Comment:
just moved this check to the top... no point to download the asset from the bucket if we don't have anywhere to send it